<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraComp</class>
 <widget class="QGroupBox" name="CameraComp">
  <property name="windowTitle">
   <string>Camera</string>
  </property>
  <property name="title">
   <string>Camera</string>
  </property>
  <layout class="QFormLayout" name="cameraLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="fieldOfViewLabel">
     <property name="text">
      <string>Field of view</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="camTypeLabel">
     <property name="text">
      <string>Camera type</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="projTypeLabel">
     <property name="text">
      <string>Projection type</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="ValuePicker" name="fieldOfView">
     <property name="minimum">
      <double>-340282346638528859811704183484516925440.000000000000000</double>
     </property>
     <property name="maximum">
      <double>340282346638528859811704183484516925440.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="camType">
     <item>
      <property name="text">
       <string>Free fly</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Look-at</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="projType">
     <item>
      <property name="text">
       <string>Perspective</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Orthographic</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ValuePicker</class>
   <extends>QDoubleSpinBox</extends>
   <header>RaZor/Interface/ValuePicker.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
